Updated readme for new developers

RetroSpaceMan123 opened the pull request:
Hello! For a class project, I was tasked to find an open source project an contribute documentation changes to it. When looking at this project, I noticed that the current readme file is not friendly to new developers, so I've decided to update it to improve the new developer experience. Thanks for reviewing this pull request!

RetroSpaceMan123 committed:
update readme

update readme such that a new developer can easily start contributing to the project

RetroSpaceMan123 committed:
Merge pull request #1 from RetroSpaceMan123/README_changes

update readme

JinayJain commented:
Hey @RetroSpaceMan123! Thanks for the contribution, really appreciate the help. This project depends on [Tauri](https://tauri.app/), which depends on Rust as well as NPM/Yarn. It'd be better if the installation instructions included the steps from (or a link to) the [prerequisites](https://tauri.app/v1/guides/getting-started/prerequisites) page.

RetroSpaceMan123 commented:
Thanks for letting me know about [Tauri](https://tauri.app/). Once I have some free time, I'll add the [prerequisites](https://tauri.app/v1/guides/getting-started/prerequisites) to the README. Once again, thanks for reviewing my pull request!

RetroSpaceMan123 committed:
Add Tauri to contribution instructions

Add the instructions to install Tauri, which is one of the dependencies for gpt-anywhere

RetroSpaceMan123 committed:
Merge pull request #2 from RetroSpaceMan123/README_changes

Add Tauri to contribution instructions

RetroSpaceMan123 commented:
Hello! I added a line in the How to Contribute section for installing Tauri. If there's anything else that needs to be changed, please let me know.

JinayJain commented:
Just one more thing from the Tauri docs, the usual development cycle would be to run `yarn tauri dev`, which should also install the relevant Rust packages. Could you add that after the `yarn install` step?
